java/integTest/src/test/java/glide/standalone/StandaloneClientTests.java

```diff
@@ -277,7 +277,6 @@ public void test_update_connection_password_connection_lost_before_password_upda
                     assertThrows(
                             ExecutionException.class,
                             () -> testClient.updateConnectionPassword(pwd, immediateAuth).get());
-            assertInstanceOf(RequestException.class, exception.getCause());
```

Comment:
I don't like. @avifenesh why it returns Closing error in CI, but Request error on other computers?